config/prod/ahayden-update-test-linv1.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
```

Comment:
this is an old template.  please use latest example, https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/3